apple/Tests/TestCases/Profile/ProfileTests.swift

```diff
@@ -5,109 +5,119 @@ import SargonUniFFI
 import XCTest
 
 final class ProfileTests: Test<Profile> {
-	
-	func test_description_and_debug() {
-		XCTAssertGreaterThan(SUT.sample.debugDescription, SUT.sample.description)
-	}
+    
+    func test_description_and_debug() {
+        XCTAssertGreaterThan(SUT.sample.debugDescription, SUT.sample.description)
```

Comment:
not your code, mine, but I think I meant to have `XCTAssertGreaterThan` be using `debugDescription.count` and `description.count`. Because it makes very little sense to compare the strings.... maybe just change to `XCTNotEqual`?

Reply:
But I think this is already tested in `Test` super class?

Reply:
@danvleju-rdx can you revert the formatting change, and just add your method? we should also merge @matiasbzurovski s Swift format PR...